.github/workflows/test_e2e_portal.yml

```
@@ -136,8 +140,13 @@ jobs:
     needs: [test-shard]
     if: ${{ always() }}
     steps:
-      - name: Required check failed
+      - name: Check required job results
         if: ${{ contains(needs.*.result, 'failure') }}
-        run: exit 1
-      - name: Dummy step
-        run: echo "All required checks have successfully passed."
+        run: |
+          echo "Some checks have failed." >> $GITHUB_STEP_SUMMARY
+          exit 1
+      - name: Check required job results
+        if: ${{ contains(needs.*.result, 'failure') != 'true' || contains(needs.*.result, 'cancelled') != 'true' || contains(needs.*.result, 'skipped') != 'true' }}
```

```suggestion
        if: ${{ contains(needs.*.result, 'failure') != 'true' || contains(needs.*.result, 'cancelled') != 'true' }}
```

I think `skipped` happens when the path-filter fails, no?